staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -535,6 +535,21 @@ const (
 	JobFailureTarget JobConditionType = "FailureTarget"
 )
 
+const (
+	// JobReasonPodFailurePolicy reason indicates a job failure condition is added due to
+	// a failed pod matching a pod failure policy rule
+	// https://kep.k8s.io/3329
+	// This is currently a beta field.
+	JobReasonPodFailurePolicy string = "PodFailurePolicy"
+	// JobReasonBackOffLimitExceeded reason indicates that pods within a job have failed a number of
+	// times higher than backOffLimit times.
+	JobReasonBackoffLimitExceeded string = "BackoffLimitExceeded"
+	// JobReasponDeadlineExceeded means job duration is past ActiveDeadline
+	JobReasonDeadlineExceeded string = "DeadlineExceeded"
+	// FailedIndexes means Job has failed indexes.
```

Comment:
This reason is used by BackoffLimitPerIndex, which is still Alpha. Given that we moved  `MaxFailedIndexesExceeded` to job controller, should not we also move this one?

Reply:
Ah. I wasn’t sure if this was part of this one or not.

Reply:
oh yes, got catch.
This is still alpha